base/src/Data/Macaw/Discovery/State.hs

```diff
@@ -240,8 +239,8 @@ emptyDiscoveryState mem addrSymMap info =
   , _globalDataMap       = Map.empty
   , _funInfo             = Map.empty
   , _unexploredFunctions = Map.empty
-  , _trustedFunctionEntryPoints = fmap (\_ -> Info.MayReturnFun) addrSymMap
-  , _exploreFnPred       = \_ -> True
+  , _trustedFunctionEntryPoints = fmap (const Info.MayReturnFun) addrSymMap
```

Comment:
I'm surprised that `hlint` didn't recommend using this instead:

```suggestion
  , _trustedFunctionEntryPoints = Info.MayReturnFun <$ addrSymMap
```

`fmap (const Info.MayReturnFun)` would work fine too, but the former is more efficient for some `Functor`s.